docs/source/tensor_view.rst

```diff
@@ -0,0 +1,71 @@
+.. currentmodule:: torch
+
+.. _tensor-view-doc:
+
+Tensor Views
+=================
+
+PyTorch allows a tensor to be `View` of an existing tensor. View tensor shares the same underlying storage
```

Comment:
"to be `View` of an existing tensor. View tensor shares" -> "to be a `View` of an existing tensor. View tensors share"

It might help to establish tensors, storage, and their relationship before explaining that tensors can share storage, and when that happens one tensor is considered the "base," and the others are considered "views."